ui/round/src/view/table.ts

```diff
@@ -64,24 +64,25 @@ const prompt = (ctrl: RoundController) => {
   };
 };
 
-export const renderTablePlay = (ctrl: RoundController) => {
+export const renderTablePlay = (ctrl: RoundController): LooseVNodes => {
   const d = ctrl.data,
     loading = isLoading(ctrl),
     { promptVNode, isQuestion } = prompt(ctrl),
     icons =
       loading || isQuestion
         ? []
         : [
-            game.abortable(d)
-              ? button.standard(ctrl, undefined, licon.X, 'abortGame', 'abort')
-              : button.standard(
-                  ctrl,
-                  d => ({ enabled: game.takebackable(d) }),
-                  licon.Back,
-                  'proposeATakeback',
-                  'takeback-yes',
-                  ctrl.takebackYes,
-                ),
+            !d.game.source.startsWith('local') &&
```

Comment:
`startsWith('local')`? What sort of `source` are we handling here?

Reply:
private games versus bots in the browser are their own game source, but it's probably better to switch on ctrl.opts.local now i think on it.

Reply:
Yes but I think that source was `local`, not `localXYZ`